packages/terra-switch/.npmignore

```diff
@@ -0,0 +1,8 @@
+*.log
+target
+reports
+tests/**/__snapshots__/
+aggregated-translations
+dev-site-config
+screenshots
+errorScreenshots
```

Comment:
"generated-themes" should also be added to this list

Reply:
added with commit : 48fce2d9ca73773ac6f9891feaa7f2a56b36f864